### ISSUE TITLE
识别训练时没有生成model日志，直接跳出

### ISSUE THREAD

**Issue by 18328401960** [PaddlePaddle/PaddleOCR#2051]
F:\PaddleOCR>python tools/train.py -c configs/rec/rec_icdar15_train.yml
[2021/02/20 17:42:08] root INFO: Architecture :
[2021/02/20 17:42:08] root INFO:     Backbone :
[2021/02/20 17:42:08] root INFO:         model_name : large
[2021/02/20 17:42:08] root INFO:         name : MobileNetV3
[2021/02/20 17:42:08] root INFO:         scale : 0.5
[2021/02/20 17:42:08] root INFO:     Head :
[2021/02/20 17:42:08] root INFO:         fc_decay : 0
[2021/02/20 17:42:08] root INFO:         name : CTCHead
[2021/02/20 17:42:08] root INFO:     Neck :
[2021/02/20 17:42:08] root INFO:         encoder_type : rnn
[2021/02/20 17:42:08] root INFO:         hidden_size : 96
[2021/02/20 17:42:08] root INFO:         name : SequenceEncoder
[2021/02/20 17:42:08] root INFO:     Transform : None
[2021/02/20 17:42:08] root INFO:     algorithm : CRNN
[2021/02/20 17:42:08] root INFO:     model_type : rec
[2021/02/20 17:42:08] root INFO: Eval :
[2021/02/20 17:42:08] root INFO:     dataset :
[2021/02/20 17:42:08] root INFO:         data_dir : ./train_data/ic15_data/
[2021/02/20 17:42:08] root INFO:         label_file_list : ['./train_data/ic15_data/rec_gt_test.txt']
[2021/02/20 17:42:08] root INFO:         name : SimpleDataSet
[2021/02/20 17:42:08] root INFO:         transforms :
[2021/02/20 17:42:08] root INFO:             DecodeImage :
[2021/02/20 17:42:08] root INFO:                 channel_first : False
[2021/02/20 17:42:08] root INFO:                 img_mode : BGR
[2021/02/20 17:42:08] root INFO:             CTCLabelEncode : None
[2021/02/20 17:42:08] root INFO:             RecResizeImg :
[2021/02/20 17:42:08] root INFO:                 image_shape : [3, 32, 100]
[2021/02/20 17:42:08] root INFO:             KeepKeys :
[2021/02/20 17:42:08] root INFO:                 keep_keys : ['image', 'label', 'length']
[2021/02/20 17:42:08] root INFO:     loader :
[2021/02/20 17:42:08] root INFO:         batch_size_per_card : 256
[2021/02/20 17:42:08] root INFO:         drop_last : False
[2021/02/20 17:42:08] root INFO:         num_workers : 4
[2021/02/20 17:42:08] root INFO:         shuffle : False
[2021/02/20 17:42:08] root INFO:         use_shared_memory : False
[2021/02/20 17:42:08] root INFO: Global :
[2021/02/20 17:42:08] root INFO:     cal_metric_during_train : True
[2021/02/20 17:42:08] root INFO:     character_dict_path : ppocr/utils/ic15_dict.txt
[2021/02/20 17:42:08] root INFO:     character_type : ch
[2021/02/20 17:42:08] root INFO:     checkpoints : None
[2021/02/20 17:42:08] root INFO:     debug : False
[2021/02/20 17:42:08] root INFO:     distributed : False
[2021/02/20 17:42:08] root INFO:     epoch_num : 72
[2021/02/20 17:42:08] root INFO:     eval_batch_step : [0, 2000]
[2021/02/20 17:42:08] root INFO:     infer_img : doc/imgs_words_en/word_10.png
[2021/02/20 17:42:08] root INFO:     infer_mode : False
[2021/02/20 17:42:08] root INFO:     log_smooth_window : 20
[2021/02/20 17:42:08] root INFO:     max_text_length : 25
[2021/02/20 17:42:08] root INFO:     pretrained_model : ./pretrain_models/rec_mv3_none_bilstm_ctc_v2.0_train/best_accuracy
[2021/02/20 17:42:08] root INFO:     print_batch_step : 10
[2021/02/20 17:42:08] root INFO:     save_epoch_step : 3
[2021/02/20 17:42:08] root INFO:     save_inference_dir : None
[2021/02/20 17:42:08] root INFO:     save_model_dir : ./output/rec/ic15/
[2021/02/20 17:42:08] root INFO:     use_gpu : False
[2021/02/20 17:42:08] root INFO:     use_space_char : False
[2021/02/20 17:42:08] root INFO:     use_visualdl : False
[2021/02/20 17:42:08] root INFO: Loss :
[2021/02/20 17:42:08] root INFO:     name : CTCLoss
[2021/02/20 17:42:08] root INFO: Metric :
[2021/02/20 17:42:08] root INFO:     main_indicator : acc
[2021/02/20 17:42:08] root INFO:     name : RecMetric
[2021/02/20 17:42:08] root INFO: Optimizer :
[2021/02/20 17:42:08] root INFO:     beta1 : 0.9
[2021/02/20 17:42:08] root INFO:     beta2 : 0.999
[2021/02/20 17:42:08] root INFO:     lr :
[2021/02/20 17:42:08] root INFO:         learning_rate : 0.0005
[2021/02/20 17:42:08] root INFO:     name : Adam
[2021/02/20 17:42:08] root INFO:     regularizer :
[2021/02/20 17:42:08] root INFO:         factor : 0
[2021/02/20 17:42:08] root INFO:         name : L2
[2021/02/20 17:42:08] root INFO: PostProcess :
[2021/02/20 17:42:08] root INFO:     name : CTCLabelDecode
[2021/02/20 17:42:08] root INFO: Train :
[2021/02/20 17:42:08] root INFO:     dataset :
[2021/02/20 17:42:08] root INFO:         data_dir : ./train_data/ic15_data/
[2021/02/20 17:42:08] root INFO:         label_file_list : ['./train_data/ic15_data/rec_gt_train.txt']
[2021/02/20 17:42:08] root INFO:         name : SimpleDataSet
[2021/02/20 17:42:08] root INFO:         transforms :
[2021/02/20 17:42:08] root INFO:             DecodeImage :
[2021/02/20 17:42:08] root INFO:                 channel_first : False
[2021/02/20 17:42:08] root INFO:                 img_mode : BGR
[2021/02/20 17:42:08] root INFO:             CTCLabelEncode : None
[2021/02/20 17:42:08] root INFO:             RecResizeImg :
[2021/02/20 17:42:08] root INFO:                 image_shape : [3, 32, 100]
[2021/02/20 17:42:08] root INFO:             KeepKeys :
[2021/02/20 17:42:08] root INFO:                 keep_keys : ['image', 'label', 'length']
[2021/02/20 17:42:08] root INFO:     loader :
[2021/02/20 17:42:08] root INFO:         batch_size_per_card : 256
[2021/02/20 17:42:08] root INFO:         drop_last : True
[2021/02/20 17:42:08] root INFO:         num_workers : 8
[2021/02/20 17:42:08] root INFO:         shuffle : True
[2021/02/20 17:42:08] root INFO:         use_shared_memory : False
[2021/02/20 17:42:08] root INFO: train with paddle 2.0.0-rc1 and device CPUPlace
[2021/02/20 17:42:08] root INFO: Initialize indexs of datasets:['./train_data/ic15_data/rec_gt_train.txt']
[2021/02/20 17:42:08] root INFO: Initialize indexs of datasets:['./train_data/ic15_data/rec_gt_test.txt']
[2021/02/20 17:42:08] root INFO: load pretrained model from ['./pretrain_models/rec_mv3_none_bilstm_ctc_v2.0_train/best_accuracy']
[2021/02/20 17:42:08] root INFO: train dataloader has 17 iters, valid dataloader has 9 iters
[2021/02/20 17:42:08] root INFO: During the training process, after the 0th iteration, an evaluation is run every 2000 iterations
[2021/02/20 17:42:08] root INFO: Initialize indexs of datasets:['./train_data/ic15_data/rec_gt_train.txt']
[2021/02/20 17:45:05] root INFO: epoch: [1/72], iter: 10, lr: 0.000500, loss: 8.828528, acc: 0.207031, norm_edit_dis: 0.695941, reader_cost: 0.04706 s, batch_cost: 17.71910 s, samples: 2816, ips: 15.89246
![image](https://user-images.githubusercontent.com/60595295/108591721-155ac380-73a5-11eb-8e62-2d74294c9c9a.png)

您好，日志到这后直接跳出，没有生成model。想询问下解决办法

**Comment by LDOUBLEV** [comment]
[2021/02/20 17:42:08] root INFO: eval_batch_step : [0, 2000]

看你的配置是这个，表示每训练2000次才评估一次

**Comment by 18328401960** [comment]
> [2021/02/20 17:42:08]根信息：eval_batch_step：[0，2000]
> 
> 看你的配置是这个，表示每训练2000次才评估一次



> [2021/02/20 17:42:08]根信息：eval_batch_step：[0，2000]
> 
> 看你的配置是这个，表示每训练2000次才评估一次

好的，谢谢。
